src/.pre-commit-config.yaml.jinja

```
@@ -101,6 +101,12 @@ repos:
       - id: oca-update-pre-commit-excluded-addons
       - id: oca-fix-manifest-website
         args: ["{{ repo_website }}"]
+      - id: oca-gen-addon-readme
+        args:
+          - --addons-dir=.
+          - --branch={{ "%.01f" | format(odoo_version) }}
+          - --repo-name={{ repo_slug }}
+          - --if-fragments-changed
```

```suggestion
          - --if-source-changed
```

Also should we add `--no-gen-html` to not generate the `index.html` reduce noise in PRs ?
In any case, the bot will take care of that and do a commit anyway we we bump the version.

Other than that this looks good to me, thanks!

---

I don't think so.

The main issue that this change is trying to fix is the fact that the module README in the runboat doesn't match the fragments when they change.

If we don't render the html, that will still happen, because Odoo displays the html readme if available. So IMHO this should stay as it is.

---

I hadn't thought about the runboat use case. Makes sense.

---

Can you just replace with `--if-source-changed` (--if-fragments-changed was confusing because it does not reflect the fact that the manifest is taken into account too).

---

And a last rebase to resolve the conflict, then good to go as far as I'm concerned.